gems/aws-eventstream/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Allow 429 response code to trigger throttle detection
```

Comment:
This would need to be in aws-sdk-core, as far as I can see you have not modified the eventstream gem.